src/lib/components/Header/ListItems.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
```

Comment:
this file was removed before, wasn't it?

Reply:
if you delete these commits, then this will lead to conflicts, so I think we should not delete them